server/item.go

```diff
@@ -82,6 +90,15 @@ func (s *RESTServer) getblob(w http.ResponseWriter, r *http.Request, id string,
 		// cache miss...load from main store, AND put into cache
 		nCacheMiss.Add(1)
 		log.Printf("Cache Miss %s", key)
+
+		// If Item Store Use disabled, and not in cache, return 503
+		if s.TapeUsed == false {
+			w.WriteHeader(503)
+			fmt.Fprintln(w, items.ErrNoStore)
```

Comment:
don't use the error message in `items`. This introduces an unnecessary coupling between the current module and the `items` package.